sprintReactMissions/src/components/ItemImage.jsx

```diff
@@ -0,0 +1,21 @@
+// 상품 fallback 이미지 컴포넌트
+import React from "react";
+import fallbackImage from "../assets/fallback.png";
+
+function ItemImage({ src, alt }) {
+  const validSrc = src && src.trim() !== "" ? src : fallbackImage;
```

Comment:
```suggestion
  const validSrc = src?.trim() || fallbackImage;
```
이렇게 해주면 복잡해보이는 조건식을 조금 단순화해줄수있겠네요 :)

Reply:
?. (optional chaining)을 사용해, src가 null/undefined인지 확인 후 trim()으로 공백 제거하고, `||(Logical OR)`로 좌변이 falsy한 값이라면 fallback 이미지를 사용하는 코드입니다.